pkg/util/uuid/uuid_wrapper.go

```diff
@@ -114,6 +114,7 @@ func FastMakeV4() UUID {
 type defaultRandReader struct{}
 
 func (r defaultRandReader) Read(p []byte) (n int, err error) {
+	//lint:ignore SA1019 deprecated
```

Comment:
Can you file a GH issue for these usages? I feel these will just be forgotten/ignored as soon as you merge the PR.

Reply:
filed: https://github.com/cockroachdb/cockroach/issues/110597

Reply:
done